spec/controllers/account/contacts_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Account::ContactsController, type: :controller do
+  describe "GET #index" do
+    it "returns a 200 status code" do
+      @super_admin = FactoryBot.create(:user, :super_admin)
+      sign_in @super_admin
+      get :index
+      expect(response).to have_http_status(200)
+    end
+  end
+  describe 'DELETE destroy' do
+    let!(:contact) { FactoryBot.create(:contact) }
+    render_views
```

Comment:
it suppose to be enabled by default

Reply:
I didn't notice that it was configured manually. Deleted.